assets/i18n/en_US.json

```diff
@@ -220,7 +222,9 @@
 
         "deleteKeystoreLabel": "Delete keystore",
         "deleteKeystoreHint": "Delete the keystore used to sign the app",
-        "deleteKeystoreDialogText": "Are you sure you want to delete the keystore used to sign patched applications?",
+
+        "deleteKeystoreDialogTitle": "Delete keystore?",
+        "deleteKeystoreDialogText": "Using new keystore will make the manager unable to update patched application that have been signed using old keystore.",
```

Comment:
This can be `Patched apps signed with the old keystore will no longer be able to update.`